examples/15_compression.cpp

```diff
@@ -0,0 +1,328 @@
+/* Copyright 2025 Franz Poeschel
```

Comment:
Do you want to split this file in a "compressor" and "plugin/advanced" compression or HDF5 plugins?

Reply:
Or at least move the advanced part of blosc down to the end?